open-ocr-2/Dockerfile

```diff
@@ -3,47 +3,57 @@ FROM ubuntu
 ENV GOPATH /opt/go
 
 RUN apt-get update && apt-get install -y \
-	autoconf \
-	automake \
-	libtool \
-	autoconf-archive \
-	pkg-config \
-	libpng12-dev \
-	libjpeg8-dev \
-	libtiff5-dev \
-	zlib1g-dev \ 
-	libicu-dev \
-	libpango1.0-dev \
-	libcairo2-dev \
-	git \
-	golang \
-	gcc \
-	curl && \
-	rm -rf /var/lib/apt/lists/*
-
-RUN curl http://www.leptonica.org/source/leptonica-1.74.1.tar.gz -o leptonica-1.74.1.tar.gz && \
-	tar -zxvf leptonica-1.74.1.tar.gz && \
-	cd leptonica-1.74.1 && ./configure && make && make install && \
-	cd .. && rm -rf leptonica*
-
-RUN git clone --depth 1 https://github.com/tesseract-ocr/tesseract.git && \
-	cd tesseract && \
-	./autogen.sh && \
-	./configure && \
-	LDFLAGS="-L/usr/local/lib" CFLAGS="-I/usr/local/include" make && \
-	make install && \
-	ldconfig && \
-	make training && \
-	make training-install && \
-	cd .. && rm -rf tesseract
-
-# Get trainnedata
-RUN git clone https://github.com/tesseract-ocr/tessdata && \
-	mv  -v tessdata/* /usr/local/share/tessdata/ && \
-	rm -rf tessadata
-
-RUN mkdir -p $GOPATH
-	
+  git \
+  golang \
+  gcc
+
+
+RUN apt-get update && apt-get install -y software-properties-common && add-apt-repository ppa:alex-p/tesseract-ocr && apt-get update

```

Comment:
Yeah, ok that makes sense.  But the comment should probably be changed to comment the "why" rather than the "what".  

Instead of:

> get the software proprerties common golang gcc and git

should be

> We need to install the software properties common before doing add-apt-repository, otherwise it will give an error: add-apt-repository: not found